noxfile.py

```diff
@@ -35,7 +35,7 @@
 def build(session):
     """Build package."""
     session.install("build", "wheel")
-    session.run("python", "-m", "build")
```

Comment:
I prefer this continue to work. We should not drop `python -m build` support as it is the new/right way to build.

Reply:
Having

```

[build-system]
requires = ["setuptools"]
build-backend = "setuptools.build_meta"
```

below should enable support for this